MapboxCoreNavigation/TunnelIntersectionManager.swift

```diff
@@ -98,8 +90,7 @@ open class TunnelIntersectionManager: NSObject {
     
     @objc public func enableTunnelAnimation(for manager: CLLocationManager,
                                         routeController: RouteController,
-                                          routeProgress: RouteProgress,
-                                               callback: RouteControllerSimulationCompletionBlock?) {
+                                          routeProgress: RouteProgress) {
         guard !isAnimationEnabled else { return }
         
         isAnimationEnabled = true
```

Comment:
@1ec5 we also need to move this down to https://github.com/mapbox/mapbox-navigation-ios/pull/1414/files#diff-987c4bbd828e654bd63d59215f890022L110

And, we need to also add `self.animatedLocationManager?.startUpdatingLocation()` it got removed here.
https://github.com/mapbox/mapbox-navigation-ios/pull/1397/files#diff-9bf14ebab2a9283285d3a8e91bd09e86L54